test/models/query_builder_test.rb

```diff
@@ -56,4 +56,16 @@ class QueryBuilderTest < ActiveSupport::TestCase
     }
     assert_equal(expected, QueryBuilder.new(search).query)
   end
+
+  test 'query builder can read TIMDEX_INDEX from env' do
```

Comment:
I see this test for the QueryBuilder, which takes care of this functionality for searches. I don't see something similar on the record side, though?

The changes to `record_controller_test.rb` are renaming cassettes (which is a good thing to do, we've talked about cassette names needing a bit more structure).

Looking at the change to the controller itself, I also don't know that a test makes sense there. I can't think of what a meaningful controller test would be, as the change is just to load a value from env (which feels like the sort of thing that we shouldn't have to test).

These tests for the query builder make sure that the built query reflects what it is told - so I think the test I'm looking for would be something testing `/app/models/timdex_record.rb` to make sure that the query it builds is responsive to changed env.

Maybe something under https://github.com/MITLibraries/timdex-ui/blob/main/test/models/timdex_test.rb#L17 ?